tests/Element/TagInlineTest.php

```
@@ -49,438 +49,429 @@ final class TagInlineTest extends TestCase
+                ->render(),
             "Failed asserting that element renders correctly with 'events()' method.",
         );
     }
 
     public function testRenderWithGlobalDefaultsAreApplied(): void
     {
-        $previous = SimpleFactory::getDefaults(TagInline::class);
+        SimpleFactory::setDefaults(
+            TagInline::class,
+            ['class' => 'from-global'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)